src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product" data-qa="card">
+      <div class="product__photo">
+        <img
+          src="images/imac.jpeg"
+          alt="iMac photo"
+          class="product__image"
+          >
+      </div>
+      <div class="product__description">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="product__code">
+        Код товара: 195434
+      </div>
+      <div class="product__rating">
+        <div class="product__stars">
+          <div class="product__stars-item"></div>
+          <div class="product__stars-item"></div>
+          <div class="product__stars-item"></div>
+          <div class="product__stars-item"></div>
+          <div class="product__stars-item"></div>
+        </div>
```

Comment:
use your code from previous task, this block should be equal to those from `stars block`